src/components/pages/blog/not-avilable-in-german.tsx

```diff
@@ -17,9 +16,7 @@ export const NotAvailableInGerman = () => {
   return (
     <div>
       <p>Der Blog ist nur auf Englisch verfügbar.</p>
-      <SendButton onClick={switchToEnglish}>
-        <ButtonText>Zum Blog</ButtonText> <RightArrowIcon />
-      </SendButton>
+      <Button onClick={switchToEnglish}>Zum Blog</Button>
```

Comment:
This is not related to the form but you got here by fixing the send button? Do we have other unrelated changes like that? You could bundle them and deliver them as an isolated refactoring.

Reply:
That was something I had to do when I was deleting the legacy form fields. The former used button was defined in the forms

Reply:
I sorted them out to another PR, should not be anymore in this one 👍